src/ontology/cl-edit.owl

```diff
@@ -29807,7 +29807,7 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023063 "MGE interneuron")
 AnnotationAssertion(rdfs:label obo:CL_4023063 "medial ganglionic eminence derived interneuron")
 EquivalentClasses(obo:CL_4023063 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002202 obo:UBERON_0004024)))
 
-# Class: obo:CL_4023064 (caudal ganglionic eminence derived interneuron)
```

Comment:
Could you keep this label as a broad or related synonym?
Also, 'CGE interneuron; shouldn't be and exact synonym, probably related.